daphne_worker/src/lib.rs

```diff
@@ -2,6 +2,22 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
 #![warn(unused_crate_dependencies)]
+#![warn(clippy::pedantic)]
+#![allow(clippy::module_name_repetitions)]
+#![allow(clippy::must_use_candidate)]
+#![allow(clippy::missing_panics_doc)]
+#![allow(clippy::missing_errors_doc)]
+#![allow(clippy::cast_precision_loss)]
+#![allow(clippy::too_many_lines)]
+#![allow(clippy::needless_pass_by_value)]
+#![allow(clippy::ignored_unit_patterns)]
+#![allow(clippy::if_not_else)]
+#![allow(clippy::default_trait_access)]
+#![allow(clippy::items_after_statements)]
+#![allow(clippy::redundant_closure_for_method_calls)]
+#![allow(clippy::inconsistent_struct_constructor)]
+#![allow(clippy::similar_names)]
+#![allow(clippy::inline_always)]
```

Comment:
This long list of allows makes me wonder if adding the aggressive clippy to CI is the right strategy. To be clear, I don't mind the changes that it suggested, it's just that I worry if this will be a huge pain. Maybe we should just run it once in a while?

Reply:
Similar comment elsewhere.